src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/jpa/PrisonerIncentiveLevel.kt

```diff
@@ -5,12 +5,14 @@ import org.springframework.data.annotation.Id
 import org.springframework.data.annotation.ReadOnlyProperty
 import org.springframework.data.annotation.Transient
 import org.springframework.data.domain.Persistable
+import org.springframework.data.relational.core.mapping.Table
 import uk.gov.justice.digital.hmpps.incentivesapi.dto.IsRealReview
 import uk.gov.justice.digital.hmpps.incentivesapi.dto.ReviewType
 import java.time.LocalDate
 import java.time.LocalDateTime
 
-data class PrisonerIepLevel(
+@Table("prisoner_iep_level")
+data class PrisonerIncentiveLevel(
```

Comment:
I'd actually probably call this `IncentiveReview` if we were to review this.

The table contains the reviews and an instance of this is a single Incentive review.

It would also make most of the code that deals with review so more meaningful 'cause prisonerIepLevels have always confused me, especially as it almost sounds like a specific Incentive level rather than a review.